benchmarks/python/conftest.py

```diff
@@ -97,6 +97,10 @@ def pytest_configure(config):
         "markers",
         "inner_persistent: mark tests using inner_persistent scheduler if not being segmented.",
     )
+    config.addinivalue_line(
+        "markers",
+        "resize: mark tests using resize scheduler if not being segmented.",
```

Comment:
@naoyam Is resize a reasonable marker for rope tests? IIUC, eventually the entire rope fwd/bwd should all go through resize scheduler.

Reply:
I think so. IIUC, we can have multiple markers, so we could have other scheduler names as well, but these benchmarks are mainly for the resize scheduler, just having `resize` seems to make sense. We could always add more later.